src/sdam/topology.ts

```diff
@@ -754,10 +753,6 @@ export class Topology extends TypedEventEmitter<TopologyEvents> {
     if (typeof callback === 'function') callback(undefined, true);
   }
 
-  get clientMetadata(): ClientMetadata {
```

Comment:
happy to put this back, although we'd have to change it to return a promise.

I could only find usages of this in tests, which I worked around by checking for the metadata on the options object instead.